pkg/storage/chunk/cache/cache.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	MemcacheClient MemcachedClientConfig `yaml:"memcached_client"`
 	Redis          RedisConfig           `yaml:"redis"`
 	Fifocache      FifoCacheConfig       `yaml:"fifocache"`
+	Groupcache     GroupConfig           `yaml:"groupcache"` // TODO(kavi): consider resolving conflict with `GroupCache` field in same struct.
```

Comment:
Is this TODO still relevant?

Reply:
Yes. we have two fields on this struct.

`Groupcache` -> GroupConfig (named it this way to be consistent with fifocache)
`GroupCache` -> Instance of groupcache we populate in `init modules.go`. (not exposed `yaml:"-"`)

Any suggestions welcome!

Reply:
Thanks for the explanation. I think we should resolve this somehow before merging

Reply:
Fixed it in https://github.com/grafana/loki/pull/6793/commits/d7df061d90ae4ee5cb61963d3c781e152a1e7ebd